Adds two tracks (#1)
kensho-technologies/career-tracks: individual_research_scientist.md

| Autonomy<ul><li>Bias to Action</li><li>Influence on Work</li><li>Resourcefulness</li><li>Delegation</li><li>Prioritization</li></ul>| [Research Scientist I](#rs1_autonomy) | [Research Scientist II ](#rs2_autonomy)  | [Senior Research Scientist I](#srs1_autonomy)  | [Senior Research Scientist II](#srs2_autonomy)  | [Staff Research Scientist](#staff_autonomy)  | 
| Teamwork <ul><li>Feedback</li><li>Communication</li><li>Participation</li><li>Go Team</li></ul> | [Research Scientist I](#rs1_teamwork) | [Research Scientist II ](#rs2_teamwork) | [Senior Research Scientist I](#srs1_teamwork) | [Senior Research Scientist II](#srs2_teamwork) | [Staff Research Scientist](#staff_teamwork) | 

<a name="rs1"></a>
## Research Scientist I

<a name="rs1_techexc"></a>
### Technical Excellence
###### [Jump to top](#main)
 
 |  |  |
 | --- | --- |
 | **Technical Knowledge** | Has a solid foundation in ML/NLP, including a working knowledge of older, non-neural approaches. Occassionally needs to consult resources for some fundamental concepts (e.g., MLE, PyTorch documentation, loss functions).|

>  | **Technical Problem Solving** | Can diagnose and fix common issues regarding ML problems and concepts; however, needs assistance and direction when results aren't progressing for weeks. Often relies on mentors and other teammembers to help unblock. Needs some supervision to ensure the work is soundly executed.|

Reviewer: ```suggestion
 | **Technical Problem Solving** | Can diagnose and fix common issues regarding ML problems and concepts; however, needs assistance and direction when results aren't progressing for weeks. Often relies on mentors and other team members to help unblock. Needs some supervision to ensure the work is soundly executed.|
```

<a name="rs2"></a>
## Research Scientist II

<a name="rs2_techexc"></a>
### **Technical Excellence**
###### [Jump to top](#main)
| | | 
| -- | -- |
 | **Technical Knowledge** | Has a strong foundation in ML/NLP, spanning traditional and modern approaches. Generally doesn't need to rely on external resources for common ML knowledge and can meaningfully discuss important ML/NLP concepts. Stays aware of some of the latest developments (e.g., PyTorch, libraries, new models or platforms).|
 | **Technical Problem Solving** | Can diagnose and fix most issues regarding ML problems, while occasionally relying on others to help diagnose issues and provide feedback on how to unblock. Can be generally trusted that the work was soundly executed without much help or guidance. For thoroughness, we will still double-check one's work and ask them for intricate details.|
 | **Research Knowledge** | Has a solid understanding of modern literature, including a working knowledge of some key works from 5-10 years ago. That is, has moderate amount research knowledge in terms of breadth and depth.|

> | **Research Abilities** | s familiar and comfortable with the process of conducting novel research and needs little assistance in identifying problems and potential approaches. Needs some assistance and feedback in balancing time spent on reading papers, brainstorming novel research ideas, writing code, and running experiments|

Reviewer: ```suggestion
| **Research Abilities** | Is familiar and comfortable with the process of conducting novel research and needs little assistance in identifying problems and potential approaches. Needs some assistance and feedback in balancing time spent on reading papers, brainstorming novel research ideas, writing code, and running experiments|
```

 | **Research Writing** | Has solid experience publishing papers (e.g., 3-6 first author papers in top-tier conferences). Can write the majority of a paper with a moderately clear narrative; however, greatly benefits from having feedback and edits. | 
 | **Modeling/Coding** | Can write effective, efficient ML code in PyTorch. Can develop and implement novel architectures without much difficulty. Runs sound, useful experiments and can make adjustments while minimally relying on others for suggestions. Asks others for help and feedback but usually not out of reliance; that is, can generally unblock oneself.| 

<a name="rs2_leadership"></a>
### Leadership
###### [Jump to top](#main)
| | | 
| -- | -- |

>  | **Intellectual Curiosity** | Exercises curiosity in all things; encourages peers to seek anrsrs. Demonstrates strong understanding of company and engineering philosophy and principles. Provides evidence and counter proposals to support ideas. |

Reviewer: ```suggestion
 | **Intellectual Curiosity** | Exercises curiosity in all things; encourages peers to seek answers. Demonstrates strong understanding of company and engineering philosophy and principles. Provides evidence and counter proposals to support ideas. |
```

> | **Innovation** | Takes appropriate amount of risk and tries new ways of doing old things. Learning to be comfortable with failure and seeks to learn from it. Shares lessons with peers and teammates.  |

Reviewer: ```suggestion
| **Innovation** | Takes an appropriate amount of risk and tries new ways of doing old things. Is learning to be comfortable with failure and seeks to learn from it. Shares lessons with peers and teammates.  |
```
